cli/args.go

```diff
@@ -69,6 +71,7 @@ func parseTerragruntOptionsFromArgs(args []string) (*options.TerragruntOptions,
 		Logger: util.CreateLogger(""),
 		RunTerragrunt: runTerragrunt,
 		Source: terraformSource,
+		UpdateSource: updateSource,
```

Comment:
Renamed everywhere to `sourceUpdate` for consistency: https://github.com/gruntwork-io/terragrunt/pull/117/commits/1bf8ed3e3a23e2be6d8bc36c32f7b49ce7eadbe3